boot.php

```diff
@@ -1247,6 +1249,7 @@ function is_friendica_app() {
 	 * @return bool Is it a known backend?
 	 */
 	function is_backend() {
+		/// @Should be made static to speedup things
```

Comment:
In main repository it is this way, non-static. Okay, I change it the proper way. :-)